test/common/index.js

```diff
@@ -310,6 +310,7 @@ let knownGlobals = [
   constructor, // Enumerable in V8 3.21.
   global,
   process,
+  regeneratorRuntime,
```

Comment:
Could we add a todo here to remove this if there is generator support in JerryScript?

Reply:
Of course